src/game/TileEngine/Explosion_Control.cc

```diff
@@ -1472,8 +1472,8 @@ static void GetRayStopInfo(UINT32 uiNewSpot, UINT8 ubDir, INT8 bLevel, BOOLEAN f
    // Have we hit things like furniture, etc?
 	 if ( Blocking != NOTHING_BLOCKING && !fTravelCostObs )
 	 {
-      // ATE: Tall things should blaock all
-      if ( bStructHeight == 4 )
+      // ATE: Tall things should blaock all; Default wall/door height is 4
+      if ( bStructHeight > 4 )
```

Comment:
where did you get the default? Is it PROFILE_Z_SIZE or some other define?

Reply:
I printed the bStructHeight in the console and red the output. Strange thing is that bStructHeight isn't consistently used in the whole code there are calculations in LOS.cc and Weapons.cc looking like this
`if (bStructHeight > 0)`
`dEndZPos = ((FLOAT) (bStructHeight + bLevel * PROFILE_Z_SIZE) - 0.5f) * HEIGHT_UNITS_PER_INDEX;`

Reply:
ok, thanks. It's a thing to eventually clean up, but not a requirement for this PR.